src/Formatter/FormatterManager.php

```diff
@@ -23,9 +24,18 @@ class FormatterManager implements FormatterManagerInterface {
   protected $plugins;
 
   /**
+   * The resource.
+   *
+   * @todo: Remove this coupling.
```

Comment:
why? I think it would be ok to have access to the resource from the formatter, no?